### DISCUSSION TITLE
Syntax colour change in Advanced Settings page

### DISCUSSION THREAD

**Discussion by partingscientist** [uBlockOrigin/uBlock-issues#2539] [General] [answered]
Similar to #2521.

1.47.3b3
![1 47 3b3](https://user-images.githubusercontent.com/115052854/224518942-0f452615-d9a0-4bfa-8592-d5ea21af9336.png)

1.47.5b6
![1 47 5b6](https://user-images.githubusercontent.com/115052854/224518954-a7876ac2-fc05-4b2b-a872-d683d9754fb0.png)

Is this intended? Thank you!

**Comment by peace2000** [comment]
1.47.3b3 is far better.

**Comment by uBlock-user** [comment, marked as answer]
https://github.com/gorhill/uBlock/commit/f4391adc5cc85ca46bc31ee09f444e963a14b8b8
